libs/application/templates/health-insurance/src/fields/ErrorModal/ErrorModal.tsx

```diff
@@ -175,20 +85,22 @@ const ErrorModal: FC<FieldBaseProps> = ({ application }) => {
                   size="default"
                   onClick={() => {
                     closeModal()
-                    history.push(`../umsoknir/${typeId}`)
+                    if (content?.buttonAction) content?.buttonAction()
```

Comment:
We should set the buttonAction to mandatory, because we can't just have it to close modal with this workaround if the buttonAction is missing :(